include/zephyr/net/wifi.h

```diff
@@ -19,9 +19,13 @@
  */
 enum wifi_security_type {
 	WIFI_SECURITY_TYPE_NONE = 0,
+	WIFI_SECURITY_TYPE_WEP,
+	WIFI_SECURITY_TYPE_WPA_PSK,
 	WIFI_SECURITY_TYPE_PSK,
 	WIFI_SECURITY_TYPE_PSK_SHA256,
 	WIFI_SECURITY_TYPE_SAE,
+	WIFI_SECURITY_TYPE_WAPI,
+	WIFI_SECURITY_TYPE_EAP,
```

Comment:
We should either specify all EAP methods or just call it Enterprise? I would suggest adding below as we do have plans to support Enterprise mode in the future.

```
WIFI_SECURITY_TYPE_EAP_PEAP
WIFI_SECURITY_TYPE_EAP_TLS
WIFI_SECURITY_TYPE_ENTERPRISE # for all others
```

Reply:
Probe response lacks this information (EAP subtype).  So including this information in scan results may not be possible.